src/main/java/at/hannibal2/skyhanni/features/garden/composter/ComposterOverlay.kt

```diff
@@ -330,6 +339,12 @@ class ComposterOverlay {
         newList.addAsSingletonList("")
     }
 
+    private val mutableMap: MutableMap<String, Double>
+        get() {
+            val baseValues = mutableMapOf<String, Double>()
+            return baseValues
+        }
```

Comment:
This seems like an entirely pointless variable.

Reply:
deleted.